ion-schema/src/isl/isl_type.rs

```diff
@@ -101,8 +99,8 @@ impl IslType {
         }
     }
 
-    pub fn name(&self) -> &Option<String> {
-        &self.name
+    pub fn name(&self) -> Option<&String> {
+        self.name.as_ref()
```

Comment:
If it's a trivial change, it'd be nice for this to return `Option<&str>` instead.